design/KR/kenward.Rmd

```diff
@@ -199,30 +185,30 @@ If a subject do not have all visits, the corresponding covariance matrix can be
 and the derivatives can be obtained through
 
 \[
-  \frac{\partial{\Sigma_i}}{\partial{\theta}} = S_i^\top \frac{\partial{\Sigma}}{\partial{\theta}} S_i
+  \frac{\partial{\Sigma_i}}{\partial{\theta_g}} = S_i^\top \frac{\partial{\Sigma}}{\partial{\theta_g}} S_i
 \]
 
 \[
-  \frac{\partial^2{\Sigma_i}}{\partial{\theta_j}\partial{\theta_k}} = S_i^\top \frac{\partial^2{\Sigma}}{\partial{\theta_j}\partial{\theta_k}} S_i
+  \frac{\partial^2{\Sigma_i}}{\partial{\theta_h}\partial{\theta_j}} = S_i^\top \frac{\partial^2{\Sigma}}{\partial{\theta_h}\partial{\theta_j}} S_i
 \]
 
-The derivative of the $\Sigma_i^{-1}$, $\frac{\partial\Sigma_i^{-1}}{\partial{\theta}}$ can be calculated through $\Sigma_i^{-1}$ and $\frac{\partial{\Sigma_i}}{\partial{\theta}}$
-using section [Derivative of the sigma inverse](#derivative-of-the-sigma-inverse).
+The derivative of the $\Sigma_i^{-1}$, $\frac{\partial\Sigma_i^{-1}}{\partial{\theta_g}}$ can be calculated through $\Sigma_i^{-1}$ and $\frac{\partial{\Sigma_i}}{\partial{\theta_g}}$
+using the above.
 
-### Scenario under grouped sigma
+### Scenario under group specific covariance estimates
 
-When conducting grouped `mmrm` models, the covariance matrix for subject i of group $g(i)$, can be written as $\Sigma_{g(i)}$.
+When fitting grouped `mmrm` models, the covariance matrix for subject i of group $g(i)$, can be written as $\Sigma_{g(i)}$.
 Assume there are $B$ groups, the number of parameters is increased by $B$ times. With the fact that for each group, the corresponding
 $\theta$ will not affect other parts, we will have block-diagonal $P$, $Q$ and $R$ matrix.
 
 \[
-P_j = \begin{pmatrix}
+P_h = \begin{pmatrix}
 P_{j, 1} & \dots & P_{j, B} \\
```

Comment:
$j$ replaced by $h$ for consistency. for the $h$ index, it is already explained [here](https://github.com/openpharma/mmrm/blob/7368e6ce65b956b9d1fc71ce69d3e55cadffc292/design/KR/kenward.Rmd#L103)

and i is for each subject because we have block diagonal matrix, see [here](https://github.com/openpharma/mmrm/blob/7368e6ce65b956b9d1fc71ce69d3e55cadffc292/design/KR/kenward.Rmd#L141) 

do we need to explain it again, or add a link?

Reply:
ok then it's fine thanks!